content/pages/exec.md

```diff
@@ -7,40 +7,29 @@ These are the people making things happen!
 
 You can view [past terms' committees here]({filename}/pages/past-exec.md).
 
-# Fall 2015 Committee #
+# Winter 2016 Committee #
 
 ## Chair ##
 
-Andee Liao (`xqliao`)
+Fatema Boxwala (`fhboxwal`)
 
-## Big CSters Director ##
-
-Jessica Yuwono (`jryuwono`)
-
-## Events Coordinator ##
-
-Ziwei Zhao (`z86zhao`)
-
-## Director of Finance ##
-
-Arshia Mufti (`a2mufti`)
-
-## Director of Publicity ##
+## Secretary ##
 
-Clare Ng (`c32ng`)
+Anna Lorimer (`aemhlori`)
 
-## Office Manager ##
+## Big CSters Directors ##
 
-Mengxue Yang (`m47yang`)
+Julia Nguyen (`j52nguye`)
+Talia McCormick(`tdmccorm`)
 
-## Secretary ##
+## Systems Administrator ##
 
-Evy Kassirer (`eawkassi`)
+Nivasini Ananthakrishnan (`nanantha`)
 
-## Systems Administrator ##
+## Administration Director ##
```

Comment:
I thought we revised this to "Social Director"? "Administration Director" isn't grammatically correct (and we already had a discussion about the title essentially having no meaning)